File: src/main/python/PageRank.py
import sys
from pyspark import SparkContext

if __name__ == "__main__":
  if len(sys.argv) != 1:
    print "Usage: PageRank"
    exit(0)
  sc = SparkContext()
  print "sc.appName = ", sc.appName
  sc.stop()

File: src/main/scala/my/spark/examples/PageRank.scala
package my.spark.examples

import org.apache.spark.SparkContext

object PageRank {
  def main(args: Array[String]) {
    if (args.length != 0) {
      println("Usage: PageRank")
      System.exit(1)
    }
    val sc = new SparkContext()
    println( "sc.appName = " + sc.appName)
    sc.stop()
  }
}
